sw/device/silicon_creator/lib/cert/cdi_1.hjson

```diff
@@ -63,6 +63,13 @@
         subject: {
             serial_number: { var: "owner_pub_key_id", convert: "lowercase-hex" },
         },
+        // From the Open Profile for DICE specification:
+        // https://pigweed.googlesource.com/open-dice/+/refs/heads/main/docs/specification.md#x_509-cdi-certificates
+        // The certificate expiration time is set to a fixed date in the future.
+        // The "not before" date is chosen to be in the past since the device does not
+        // have a reliable way to get the time.
+        not_before: "20230101000000Z",
```

Comment:
Just curious: how did you select the "not_before" value? Looks like the [open profile](https://pigweed.googlesource.com/open-dice/+/refs/heads/main/docs/specification.md?pli=1#x_509-cdi-certificates) had a different value (which corresponded to the date of publication of the TCG DICE spec. Not that it matters, just was curious :)

Reply:
The DICE specification said any date in the past is acceptable so I just took a random date. But it's true that the publication date of the spec makes the most sense ^^ I am happy to change if you think that's worth it, it's now it's trivial to change in the hjson template.

Reply:
no need to change, was just curious